### PR TITLE
Fix author

### PR DESCRIPTION
**Pull request by jules2689**
Not all commits have actual authors. If the user doesn't have an associated github account, we have no author.

e.g. capistrano/capistrano@bd56f33

This commit has no author object and so `author` is nil

### CODE REVIEW

**Comment by jules2689** [comment]
re-ping @tarebyte

File: test/fixtures/files/commit_no_author.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
+{
```
Comment (tarebyte) [inline test/fixtures/files/commit_no_author.json]: @jules2689 it actually looks like this response isn't correct, see your commit example here: https://api.github.com/repos/capistrano/capistrano/commits/bd56f33c750952770d717537d9652c6ca8e74f45

```json
{
  "sha": "bd56f33c750952770d717537d9652c6ca8e74f45",
  "commit": {
    "author": {
      "name": "Jan Ulrich",
      "email": "jan@optemo.com",
      "date": "2013-01-29T23:03:54Z"
    },
    "committer": {
      "name": "Jan Ulrich",
      "email": "jan@optemo.com",
      "date": "2013-01-29T23:06:06Z"
    },
    "message": "Update both asset mtime and atime with touch",
    "tree": {
      "sha": "8c928b84c31f90f1bab5d3caf12002dae085362e",
      "url": "https://api.github.com/repos/capistrano/capistrano/git/trees/8c928b84c31f90f1bab5d3caf12002dae085362e"
    },
    "url": "https://api.github.com/repos/capistrano/capistrano/git/commits/bd56f33c750952770d717537d9652c6ca8e74f45",
    "comment_count": 0,
    "verification": {
      "verified": false,
      "reason": "unsigned",
      "signature": null,
      "payload": null
    }
  },
  "url": "https://api.github.com/repos/capistrano/capistrano/commits/bd56f33c750952770d717537d9652c6ca8e74f45",
  "html_url": "https://github.com/capistrano/capistrano/commit/bd56f33c750952770d717537d9652c6ca8e74f45",
  "comments_url": "https://api.github.com/repos/capistrano/capistrano/commits/bd56f33c750952770d717537d9652c6ca8e74f45/comments",
  "author": null,
  "committer": null,
  "parents": [
    {
      "sha": "ea5b03b4d38eeb296778a809d927297065624981",
      "url": "https://api.github.com/repos/capistrano/capistrano/commits/ea5b03b4d38eeb296778a809d927297065624981",
      "html_url": "https://github.com/capistrano/capistrano/commit/ea5b03b4d38eeb296778a809d927297065624981"
    }
  ],
  "stats": {
    "total": 2,
    "additions": 1,
    "deletions": 1
  },
  "files": [
    {
      "sha": "1671c415eef3055105d7fe31cc147cefff9ac765",
      "filename": "lib/capistrano/recipes/deploy/assets.rb",
      "status": "modified",
      "additions": 1,
      "deletions": 1,
      "changes": 2,
      "blob_url": "https://github.com/capistrano/capistrano/blob/bd56f33c750952770d717537d9652c6ca8e74f45/lib/capistrano/recipes/deploy/assets.rb",
      "raw_url": "https://github.com/capistrano/capistrano/raw/bd56f33c750952770d717537d9652c6ca8e74f45/lib/capistrano/recipes/deploy/assets.rb",
      "contents_url": "https://api.github.com/repos/capistrano/capistrano/contents/lib/capistrano/recipes/deploy/assets.rb?ref=bd56f33c750952770d717537d9652c6ca8e74f45",
      "patch": "@@ -66,7 +66,7 @@\n         put current_assets.map{|a| \"#{shared_path}/assets/#{a}\" }.join(\"\\n\"), \"#{deploy_to}/TOUCH_ASSETS\"\n         run <<-CMD.compact\n           cat #{deploy_to.shellescape}/TOUCH_ASSETS | while read asset; do\n-            touch -cm -- \"$asset\";\n+            touch -c -- \"$asset\";\n           done &&\n           rm -f -- #{deploy_to.shellescape}/TOUCH_ASSETS\n         CMD"
    }
  ]
}
```

Comment (jules2689) [reply test/fixtures/files/commit_no_author.json]: oh interesting, I copied it from https://developer.github.com/v3/repos/commits/#get-a-single-commit

Comment (jules2689) [reply test/fixtures/files/commit_no_author.json]: Should be updated now. Used what you posted as a base

**Comment by chrisarcand** [comment]
@jules2689 ℹ️ Conflict here

**Comment by jules2689** [comment]
@chrisarcand updated

**Comment by andrew** [comment]
LGTM 👍

**Pull request merged**